docs/reference/dynamic_scenarios.rst

```diff
@@ -65,7 +65,7 @@ In detail, a single time step of a dynamic simulation is executed according to t
 
 9. Update every :term:`dynamic property` of every object to its current value in the simulator.
 
-10. If the simulation is stopping for one of the reasons above, first check if any of the :keyword:`require eventually` conditions of any remaining scenarios were never satisfied: if so, reject the simulation.
+10. If the simulation is stopping for one of the reasons above, first check if any of the liveness properties of any remaining scenarios were never satisfied: if so, reject the simulation.
```

Comment:
I didn't understand what this meant but systematically replaced "require eventually conditions" with "liveness properties". I want to make sure this makes sense.

Reply:
Makes sense to me! A liveness property is something that must occur in an execution, so "require eventually" is how we implement such a property.

Reply:
We should be careful here: a property like `(always X) and (eventually Y)` is neither a safety nor a liveness property (violations of it don't necessarily have a finite prefix after which the property can't be satisfied, nor is it always possible to extend a finite trace to one where the property is satisfied), but it needs to be checked at the end of the simulation. I'll revise this text.